pkg/glob/glob.go

```diff
@@ -40,7 +41,7 @@ func (p Pattern) Glob(cb func(PathInfo) bool) bool {
 		segs = segs[1:]
 		dir += "/"
 	} else if runtime.GOOS == "windows" && len(segs) > 1 && IsLiteral(segs[0]) && IsSlash(segs[1]) {
-		// TODO: Handle UNC.
+		// TODO: Handle Windows UNC paths. Google "windows unc path".
```

Comment:
IMO clarifying UNC to the more Google-able "Windows UNC paths" is sufficient; if the reader is not familiar with the concept they should know how to search for that themselves. Otherwise we'd litter the codebase with comments that say "Google $concept_x", which is unnecessary and a bit patronizing TBH.

Reply:
Agreed. I added that because I was surprised **I* had to google it.